.travis.yml

```diff
@@ -37,8 +37,6 @@ addons:
 
 # Install various dependencies
 install:
-  - pip install --upgrade pytest
-  - pip install --upgrade pytest-cov
```

Comment:
I have mixed feelings about adding `pytest` and `pytest-cov` in the externals. It's true that this way things will work out of the box, but:
- they are only needed for tests and I think every frequent contributor knows how to `pip` install something
- we'll need to update externals from time to time not to get stuck with old versions of the software

In any case I would leave the two lines above. This will let us work always with the latest release, and will expose any incompatibility with it as early as possible.